cypress/e2e/tests/footer.cy.js

```diff
@@ -0,0 +1,220 @@
+context('Footer Get Started', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('div')
+      .contains('Get Started')
+      .siblings('div')
+      .as('getStarted')
+  })
+
+describe('Voice', () => {
+  it('Should verify the external link exists', () => {
+    cy.get('@getStarted')
+      .contains('Voice')
+      .should('have.attr', 'href', 'https://www.bandwidth.com/voice/')
+    })
+  })
+
+describe('Messaging', () => {
+  it('Should verify the external link exists', () => {
+    cy.get('@getStarted')
+      .contains('Messaging')
+      .should('have.attr', 'href', 'https://www.bandwidth.com/messaging/')
+    })
+  })
+
+describe('Enterprise 911', () => {
+  it('Should verify the external link exists', () => {
+    cy.get('@getStarted')
```

Comment:
**Food for thought**: Room for refactoring and remove duplication? Each Test has almost identical validations. Can it be extracted?